API.md

```diff
@@ -152,3 +152,63 @@ The payload should be a classifier object, with a classifier type selector `name
 An `HTTP 404 Not Found` status code is returned if the survey with the specified ID could not be found.
 
 An `HTTP 409 Conflict` status code is returned if a classifier type selector already exists for any of the names in the payload.
+
+## Post New Survey
+* `POST /surveys` will create a new survey.
+
+The payload should be a JSON document, with an `id`, a `shortName`, a `longName`, a `surveyRef`, a `legalBasis`, a `surveyType`, and a `legalBasisRef` as strings, and `classifiers,omitempty` as a list.
+
+### Example JSON payload
+```json
+{
+    "id": "efa868fb-fb80-44c7-9f33-d6800a17c4da",
+    "shortName": "test-short-name", 
+    "longName": "test-long-name",
+    "surveyRef": "456",
+    "legalBasis": "Statistics of Trade Act 1947",
+    "surveyType": "Social",
+    "legalBasisRef": "STA1947",
+    "classifiers,omitempty": [
```

Comment:
`omitempty` is an annotation for the golang json marshaller which means to not create that entity of it is not present. essentially stating its optionality.
Basically, this doesn't form part of the payload. It just highlights that its optional